core/spec/models/spree/store_spec.rb

```diff
@@ -50,4 +50,14 @@
       expect(store.default).not_to be true
     end
   end
+
+  describe '#orders' do
```

Comment:
I'm pro adding an association for this. But I don't think we need to test rails here. We're not really testing Solidus.